libs/components/theme/src/lib/styles/_switch.scss

```diff
@@ -35,7 +35,7 @@ $sky-switch-size: 22px !default;
   -webkit-appearance: none;
 
   &:checked + .sky-switch-control,
-  &:indeterminate + .sky-switch-control {
+  &[type='checkbox']:indeterminate + .sky-switch-control {
```

Comment:
Shouldn't we have a style for indeterminate radio buttons as well? The "switch" styles are meant to be unaware of the input type so that they can be applied to any component that operates like an interactive switch. Maybe @Blackbaud-AdamFunderburk can weigh in.